jjwt/pom.xml

```diff
@@ -30,6 +30,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-security</artifactId>
         </dependency>
+        
+        <dependency>
```

Comment:
is it needed? when we used boot-1 was it included as part of any other library and with boot-2 we have to mention it explicitly?

Reply:
I added the spring-boot-starter-web explicitly since without this dependency, we are getting compile errors of the kind javax.servlet.http.* cannot be resolved.